js/index.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+
+	var cityAustin = true if 
+	var cityLA = true
+	var citySF = true
+	var citySydney = true
+
+	$('#city-type').submit(addEntry)
+
+	function addEntry (event) {
+		event.preventDefault();
+	
+	var userInput = $('#newEntry').val();
```

Comment:
You got the right idea here, just targeting the wrong element.

The idea is you'll enter your `addEntry()` handler function whenever the user submits the form. Once the user has submitted the form, break down step by step what it is you need to do. Think of it in these terms:

1. What information do I need to know from the user (the HTML)?
2. How do I get this information from the elements on the HTML page?
3. Where should I store this information the user entered?
4. What type of data do I expect the user to have entered into the HTML page? Is it a string? A number? a boolean?
5. Once I have the information stored in a variable and I know its 'type', what values should I compare my variable to?
6. If I find a match, what is it that I need to the DOM (HTML) to make the necessary change?